openapi/weaviate/openapi.yaml

```diff
@@ -0,0 +1,1801 @@
+          type: array
+        deprecations:
+          type: array
+          items:
+            $ref: '#/components/schemas/Deprecation'
+          nullable: true
+          description: deprecations 
+        totalResults:
+          description: >-
+            The total number of Objects for the query. The number of items in a
+            response may be smaller due to paging.
+          format: int64
+          type: integer
+      type: object
+    WhereFilter:
+      description: Filter search results using a where filter
+      properties:
+        operands:
+          description: combine multiple where filters, requires 'And' or 'Or' operator
+          type: array
+          items:
+            $ref: '#/components/schemas/WhereFilter'
+          nullable: true
+        operator:
+          description: operator to use
+          type: string
+          enum:
+            - And
+            - Or
+            - Equal
+            - Like
+            - Not
+            - NotEqual
+            - GreaterThan
+            - GreaterThanEqual
+            - LessThan
+            - LessThanEqual
+            - WithinGeoRange
+            - IsNull
+          example: GreaterThanEqual
+          nullable: true
+        path:
+          description: path to the property currently being filtered
+          type: array
+          items:
+            type: string
+          example:
+            - inCity
+            - City
+            - name
+          nullable: true
+        valueInt:
+          description: value as integer
+          type: integer
+          format: int64
+          example: 2000
+          nullable: true
+        valueNumber:
+          description: value as number/float
+          type: number
+          format: float
+          example: 3.14
+          nullable: true
+        valueBoolean:
+          description: value as boolean
+          type: boolean
+          example: false
+          nullable: true
+        valueString:
+          description: value as string
+          type: string
+          example: my search term
+          nullable: true
+        valueText:
+          description: value as text (on text props)
+          type: string
+          example: my search term
+          nullable: true
+        valueDate:
+          description: value as date (as string)
+          type: string
+          example: TODO
+          nullable: true
+        valueGeoRange:
+          $ref: '#/components/schemas/WhereFilterGeoRange'
+      type: object
+    WhereFilterGeoRange:
+      type: object
+      description: filter within a distance of a georange
+      properties:
+        geoCoordinates:
+          $ref: '#/components/schemas/GeoCoordinates'
+        distance:
+          type: object
+          properties:
+            max:
+              type: number
+              format: float
+              description: max distance 
+          description: distace based on geoCoordinates 
```

Comment:
Add new line

Reply:
Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/754/commits/4011e12c5729754ff4de72e28289bb1362275fd0.